tests/utils/client.py

```diff
@@ -1,6 +1,6 @@
 from steamship import Configuration, Steamship
 
-TESTING_PROFILE = "test"
+TESTING_PROFILE = "prod"
```

Comment:
This should not be in the PR right? 

I feel like we should take the profile out of test entirely to prevent this sort of stuff from slipping through